src/config/lang/english.go

```diff
@@ -260,6 +260,7 @@ const (
 	CmdPackageDeployFlagShasum                         = "Shasum of the package to deploy. Required if deploying a remote package and \"--insecure\" is not provided"
 	CmdPackageDeployFlagSget                           = "[Deprecated] Path to public sget key file for remote packages signed via cosign. This flag will be removed in v1.0.0 please use the --key flag instead."
 	CmdPackageDeployFlagPublicKey                      = "Path to public key file for validating signed packages"
+	CmdPackageDeployFlagSkipWebhooks                   = "Skip waiting for external webhooks to execute as each package component is deployed"
```

Comment:
@Racer159 Should command flags also receive `[ALPHA]`?

Reply:
Yes - good catch

Reply:
Added `[alpha]` as a prefix to the flag description